### PR TITLE
#1453 - Document metadata panel shows disabled layers

### PR DESCRIPTION
**Pull request by wpmp**
**What's in the PR**
- fix handling for disabled layers for document metadata panel

**How to test manually**
1. Have a disabled document annotation layer
2. Open Document Metadata Panel
3. Click dropdown to select a layer
4. Disabled layer should'nt show up

Also already created annotatios in afterwards disabled layers should not show up anymore

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

File: inception-layer-docmetadata/src/main/java/de/tudarmstadt/ukp/inception/ui/core/docanno/sidebar/DocumentMetadataAnnotationSelectionPanel.java [hunk 1]
```diff
@@ -254,7 +255,8 @@ protected void populateItem(ListItem<AnnotationListItem> aItem)
         
         List<AnnotationListItem> items = new ArrayList<>();
         for (AnnotationLayer layer : listMetadataLayers()) {
-            if (!DocumentMetadataLayerSupport.TYPE.equals(layer.getType())) {
+            if (!DocumentMetadataLayerSupport.TYPE.equals(layer.getType())
```
Comment (reckart) [inline inception-layer-docmetadata/src/main/java/de/tudarmstadt/ukp/inception/ui/core/docanno/sidebar/DocumentMetadataAnnotationSelectionPanel.java]: If the layers are already filtered in `listMetadataLayers()`, why filter them again?

Comment (wpmp) [reply inception-layer-docmetadata/src/main/java/de/tudarmstadt/ukp/inception/ui/core/docanno/sidebar/DocumentMetadataAnnotationSelectionPanel.java]: I wondered about this, too.
I'll remove the whole if construct

Comment (wpmp) [reply inception-layer-docmetadata/src/main/java/de/tudarmstadt/ukp/inception/ui/core/docanno/sidebar/DocumentMetadataAnnotationSelectionPanel.java]: done

**Comment by wpmp** [comment]
Jenkins, can you test this please?

**Comment by wpmp** [comment]
Jenkins, can you test this please?

**Pull request merged**